### PR TITLE
docs: Update space-before-blocks.md

### PR DESCRIPTION
**Pull request by muraha**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
fixed wrong rule param in the example

### CODE REVIEW

File: docs/rules/space-before-blocks.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The default is `"always"`.
 Examples of **incorrect** code for this rule with the "always" option:
 
 ```js
-/*eslint space-before-blocks: "error"*/
+/*eslint space-before-blocks: "always"*/
```
Comment (aladdin-add) [inline docs/rules/space-before-blocks.md:32]: I think it's working as expected - the default is "always"

**Review by aladdin-add — CHANGES_REQUESTED** [review]

File: docs/rules/space-before-blocks.md (hunk 1 above)
Comment (muraha) [reply docs/rules/space-before-blocks.md:32]: the Example block is about 'always' but inside block 'error'. it's just wording correction

Comment (aladdin-add) [reply docs/rules/space-before-blocks.md:32]: the "error/warn/off" is a level, see the docs: https://eslint.org/docs/user-guide/configuring/rules#configuring-rules

Comment (muraha) [reply docs/rules/space-before-blocks.md:32]: ah, my bad)

**Pull request closed without merging**